templates/index.phtml

```diff
@@ -24,5 +24,6 @@
                 <span id="submitWarning" class="req"></span>
             </div>
         </form>
+        <div id="error-message"></div>
```

Comment:
if it pertains to the form i'd put this div inside the form

Reply:
done